src/core/core.js

```diff
@@ -261,14 +256,14 @@ var Page = (function PageClosure() {
       var annotations = this.annotations;
       var annotationsData = [];
       for (var i = 0, n = annotations.length; i < n; ++i) {
-        annotationsData.push(annotations[i].getData());
+        annotationsData.push(annotations[i].data);
       }
       return annotationsData;
     },
 
     get annotations() {
       var annotations = [];
-      var annotationRefs = (this.annotationRefs || []);
+      var annotationRefs = (this.getInheritedPageProp('Annots') || []);
```

Comment:
While you're at it, remove the parentheses. They are not required, and do not enhance the readability of this line.